docs/tutorials/FlowTutorial.md

```diff
@@ -0,0 +1,1581 @@
+Total                  1.48e-02   1.77e-02   6.57e-08   3.25e-02 100.0%
+                          45.6%      54.4%       0.0%
+```
+
+## OpenROAD GUI
+
+The GUI allows user to select, control, highlight and navigate across the
+design hierarchy and design objects (nets, pins, instances, paths etc.)
+through detailed visualization and customization options. Find details
+on how to use the GUI [here](../../src/gui/README.md)
+
+In this section learn how to:
+
+1.  Visualize design hierarchy
+2.  Load DEF and LEF files for floorplan and layout visualization
+3.  Trace the synthesized clock tree to view hierarchy and buffers
+4.  Use heat maps to view congestion and observe the effect of placement
+5.  View and trace critical timing paths
+6.  Set display control options
+
+If you have completed the RTL-GDS flow, then proceed to view the final
+GDS file under results directory `./results/sky130hd/ibex/base/`
+
+For the `ibex` design uncomment the `DESIGN_CONFIG` variable
+in the `Makefile` available [here](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/Makefile)
+
+```
+# DESIGN_CONFIG=./designs/sky130hd/gcd/config.mk
+DESIGN_CONFIG=./designs/sky130hd/ibex/config.mk
+# DESIGN_CONFIG=./designs/sky130hd/aes/config.mk
+```
+
+```
+make gui_final
+```
+
+### Viewing Layout Results
+
+The `make gui_final` command target successively reads and loads the
+technology `.lef`, design `.def` files and the parasitics and invokes the
+GUI in these steps:
+
+-   Reads and loads `.lef` files (tech and merged)
+-   Reads and loads `.def` files (final design def)
+-   Loads `.spef` (parasitics)
+
+The figure below shows the post-routed DEF for the `ibex` design.
+
+![ibex_fina_def](./images/ibex_final_def.png)
+
+### Visualizing Design Objects And Connectivity
+
+Note the `Display Control` window on the LHS that shows buttons
+for color, visibility and selection options for various design
+objects: Layers, Nets, Instances,Blockages, Heatmaps etc.
+
+The Inspector window on the RHS allows user to inspect details of
+selected design objects and the timing report.
+
+Try selectively displaying (show/hide) various design objects through
+the display control window and observing their impact on the display.
+
+### Tracing The Clock Tree
+
+To view the synthesized clock tree for `ibex` design follow steps:
+-   From the top Toolbar Click `View–>Find`
+-   In the Dialog Box `Find Object ` search the clock net using keyword as follows:
+
+![cts_find_obect](./images/cts_find_object.png)
+
+Click `Ok` to view the synthesized clock tree of your design.
+
+To view clock tree structure below, user need to disable the Metal
+`Layers` section on LHS as shown below.
+
+![ibex_clock_tree](./images/ibex_clock_tree.png)
+
+From the top Toolbar, Click on  `Windows –> options`
+to select/hide different view options of Scripting, Display control etc.
+
+### Using Heat Maps
+
+From the Menu Bar, Click on `Tools→Heat Maps → Placement Density` to view

```

Comment:
Some places you use arrows (like here) and others you use `->`, probably should be consistent about that.
Also there should be a space between Tools and the arrow and the arrow and Heat for consistency.